src/plugins/links/public/components/links_strings.ts

```diff
@@ -22,13 +22,15 @@ export const LinksStrings = {
       i18n.translate('links.editor.updateButtonLabel', {
         defaultMessage: 'Update link',
       }),
-    getEditLinkTitle: () =>
-      i18n.translate('links.editor.editLinkTitle', {
-        defaultMessage: 'Edit link',
```

Comment:
super nit: If there is no label, will this label return two spaces between "Edit" and "link" e.g. `Edit  Link`?

Reply:
Oh cool. 👍